### PR TITLE
Documentation: install git lfs and basic PostreSQL configuration

### PR DESCRIPTION
**Pull request by socketbox**
### Summary

1. The command `git lfs install` needs to be run within a directory that contains a git repository. This is a minor correction to the docs to make sure that the user is instructed to change directories after cloning and before running `git lfs install`.

2. There didn't seem to be any information about how to go about setting up a PostgreSQL database (at least, not under the Databases section), so I added information regarding `options.ini` to that portion of the docs.

### Reviewer guidance
I tested these changes by walking through what's described in the [Getting Started](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html) section of the development docs in a pristine container.

### References
#6545

----

### Contributor Checklist

PR process:

- [?] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by nucleogenesis — APPROVED** [review]
Love the additional documentation here! Thank you @socketbox!

File: docs/getting_started.rst [hunk 1]
```diff
@@ -365,6 +365,7 @@ You can initialize the server using:
 
   kolibri manage migrate
 
+To use a PostgreSQL database, you would edit the `options.ini` file in `KOLIBRI_HOME`, setting options under the `[Database]` section to appropriate values. For the `DATABASE_ENGINE` option, use the value `postgres`.
```
Comment (nucleogenesis) [inline docs/getting_started.rst:368]: I think that we may want to hold off on adding this kind of Postgres documentation until we are able to produce more thorough documentation like the following:

https://github.com/learningequality/kolibri/issues/6480#issuecomment-583299221

I suggest that Kolibri + psql documentation should link to OS-specific installation instructions and provide universal details for how to setup proper roles, databases, etc to use psql with Kolibri. 

If we mention PostgreSQL only briefly, then I think we should note to users that we recommend PostgreSQL only for those who already familiar with setting it up - in which case I think that this paragraph would be very helpful for someone trying to figure out how/where to configure their database.

**Review by nucleogenesis — CHANGES_REQUESTED** [review]
I approved initially but thought a bit more on the PostgreSQL bit that @socketbox added and realize that maybe we should modify the existing text slightly (eg "...PostgreSQL only suggested for experienced users...") -- but also that in any case we may do well to provide more thorough set of docs targeted for less experienced devs.

In any case - I'll tag @rtibbles  and @jonboiser for their thoughts as well. Thanks a ton for the submission @socketbox !

**Comment by jonboiser** [comment]
Closing for now. We'll need to consolidate all the notes from Slack, GH, etc. to make a more comprehensive and current story for using postgres with Kolibri.

**Pull request closed without merging**